schemas/sublime-theme.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Please note, that sublime-theme files are not neccessarily "array" objects. They can be "objects" as well, since ST3.2

Reply:
Thanks, fixed.